docs(Toolbar): add new adaptive parameter (#2694)
telerik/blazor-docs: components/toolbar/overview.md

| Parameter | Type | Description |
| ----------- | ----------- | ----------- |

> | `Adaptive ` | `bool` <br /> (`true`) | Toggles the overflow popup of the ToolBar. The component displays an additional anchor on its side, where it places all items which do not fit and overflow. [Template items](slug://toolbar-templated-item#notes) don't participate in this mechanism and they are always rendered in the ToolBar itself. **This parameter will become obsolete in the next major version in favor of the `OverflowMode` parameter. Both parameters will be available until then.** |

Reviewer: See how we documented such information in the past:
https://github.com/telerik/blazor-docs/blob/e6b3de192b520ba2ce302c35a0833b534c41f046/components/textarea/overview.md#textarea-parameters

```suggestion
| `Adaptive` <br /> (deprecated) | `bool` <br /> (`true`) | Toggles the overflow popup of the ToolBar. The component displays an additional anchor on its side, where it places all items which do not fit and overflow. [Template items](slug://toolbar-templated-item#notes) don't participate in this mechanism and they are always rendered in the ToolBar itself. This parameter is deprecated in favor of the `OverflowMode` parameter.** |
```
